locale/it_IT/default.xml

```diff
@@ -145,4 +140,30 @@ Il paragrafo che comincia con "Questo ultimo argomento..." non è allineato</pre
 	<message key="section.default.policy"></message>
 	<message key="default.journalSettings.purchaseIssueFee">Acquista il fascicolo</message>
 	<message key="default.journalSettings.purchaseIssueFeeDescription">Il pagamento di questa quota ti permetterà di visualizzare, scaricare, e stampare questo fascicolo.</message>
+	<message key="default.genres.article">Articolo</message>
+	<message key="default.groups.name.manager">Direttore</message>
+	<message key="default.groups.plural.manager">Direttori</message>
+	<message key="default.groups.abbrev.manager">MNGR</message>
+	<message key="default.groups.name.editor">Editor</message>
+	<message key="default.groups.plural.editor">Editor</message>
+	<message key="default.groups.abbrev.editor">EDT</message>
+	<message key="default.groups.name.guestEditor">Guest editor</message>
```

Comment:
I know... but the translator put it in there, it's not left over from an English file, so I thought I'd keep it…